server/tserver/src/main/java/org/apache/accumulo/tserver/log/RecoveryLogReader.java

```diff
@@ -127,6 +127,9 @@ public RecoveryLogReader(VolumeManager fs, Path directory, LogFileKey start, Log
         foundFinish = true;
         continue;
       }
+      if (SortedLogState.FAILED.getMarker().equals(child.getPath().getName())) {
```

Comment:
Could add this condition as an OR to the first if stmt in the for loop.

Reply:
I was thinking this is easier to remove, if that's what we plan on for 2.0.